### ISSUE TITLE
Windows output directory name is invalid 

### ISSUE THREAD

**Issue by AndersonHappens** [javascript-obfuscator/javascript-obfuscator#156]
In windows, if you try to run "javascript-obfuscator . " then it will create a directory with the obfuscated files with the name "-obfuscated.", which is invalid in windows. This happens even if you specify the output directory like:  "javascript-obfuscator . --output ./obfuscated" 

Turns out, you can create a directory that ends with "." fine, but any attempt to access, delete, rename, etc always gives either the error "Windows cannot find this file" or "this directory is unavailable". The only way to resolve this seems to be by using 7-zips file manager to rename the folder manually.

**Comment by sanex3339** [comment]
I'll try to reproduce it today

**Comment by sanex3339** [comment]
Can reproduce on my mac

**Comment by sanex3339** [comment]
Will fixed in 0.14.0 somewhere in january.

Here some examples from updated readme:
```sh
javascript-obfuscator ./dist [options]
// creates a new obfuscated files under `./dist` directory near the input files with `obfuscated` postfix

javascript-obfuscator ./dist --output ./dist/obfuscated [options]
// creates a folder structure with obfuscated files under `./dist/obfuscated` path
```

**Comment by sanex3339** [comment]
out in `javascript-obfuscator@0.14.0`
